meta-networking/recipes-support/strongswan/strongswan_5.6.2.bb

```diff
@@ -10,6 +10,7 @@ DEPENDS = "gmp openssl flex-native flex bison-native"
 SRC_URI = "http://download.strongswan.org/strongswan-${PV}.tar.bz2 \
            file://fix-funtion-parameter.patch \
            file://0001-memory.h-Include-stdint.h-for-uintptr_t.patch \
+           file://ipsec.init \
```

Comment:
The sysvinit stuff and the ipsec init would be better as a meta-nilrt patch, I think.

Reply:
got it.